kyo-core/shared/src/main/scala/kyo/aborts.scala

```diff
@@ -13,8 +13,8 @@ object Aborts:
 
     extension [V](self: Aborts[V])
 
-        def fail[T](value: T)(using ev: T => V, t: Tag[Aborts[V]]): Nothing < Aborts[V] =
-            self.suspend[Nothing](Left(ev(value)))
+        def fail[T <: V](value: T)(using t: Tag[Aborts[T]]): Nothing < Aborts[T] =
```

Comment:
I've changed this method to suspend the specific type of the parameter. For example `Aborts[Throwable].fail(new Exception` will now return a `T < Aborts[Exception]`.